tools/autograph-monitor/contentsignaturepki_test.go

```diff
@@ -109,8 +109,8 @@ func signTestCert(options signOptions) *x509.Certificate {
 	return certX509
 }
 
-func sha2Fingerprint(cert *x509.Certificate) string {
-	return strings.ToUpper(fmt.Sprintf("%x", sha256.Sum256(cert.Raw)))
+func sha2Fingerprint(cert *x509.Certificate) []string {
```

Comment:
I understand turning this into a `[]string` return type is convenient, but this is not what I would expect to receive from a func taking a single `cert` and would spend time investigating why. Could we do this outside of this func (and its similarly named friend elsewhere) or change this (and its similarly named friend) to take multiple certs?

Reply:
Yeah that makes sense. Adjusting this and fixed the property names too.